barretenberg/cpp/scripts/audit/run_dashboard_server.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+# Resolve script directory
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+summary_script="$script_dir/generate_audit_summary.py"
+dashboard_html="$script_dir/audit_dashboard.html"
+
+# Check that dashboard HTML exists
```

Comment:
I would have probably been tempted to just generate static HTML that gets posted to a github page (e.g. `https://aztecprotocol.github.io/aztec-packages/dev/audit`)

Reply:
Fine with any implementation to break the ice here

Reply:
Definitely.